ui/app/src/components/PublishDialog/PublishDialogContainer.tsx

```diff
@@ -173,13 +175,13 @@ export function PublishDialogContainer(props: PublishDialogContainerProps) {
 
   const publishSource = useMemo(
     () => ({
-      items,
+      items: !detailedItems.isFetching ? Object.values(detailedItems.itemsByPath) : null,
       error: state.error,
       submitting: isSubmitting,
       publishingTargets,
       myPermissions
     }),
-    [items, state.error, isSubmitting, publishingTargets, myPermissions]
+    [detailedItems, state.error, isSubmitting, publishingTargets, myPermissions]
```

Comment:
The return value of `useDetailedItems` is a new object reference every time. This memo will renew every render.